create_plugin.py

```diff
@@ -10,6 +10,10 @@ def checkValidPlugin(path):
     # A plugin must contain an __init__.py
     if not os.path.isfile(os.path.join(path, "__init__.py")):
         return False
+    
+    # A plugin must contain an __init__.py
```

Comment:
This comment doen't seem to be relevant to `plugin.json` 😇

Reply:
Ah, damn. Copy/pasted the section and forgot to correct the comment, too.
My finger was to early on the [commit] button :stuck_out_tongue_closed_eyes: